### ISSUE TITLE
Code question

### ISSUE THREAD

**Issue by Summer-Canvas** [Ryanvuha/DRL_beam_hopping_precoding_GEO#1]
Could please provide me with the paper infomation related to this repo？

**Comment by laomaoqwert** [comment]
Hi, Have you solved these problems,and did your code run successfully?

**Comment by Summer-Canvas** [comment]
no. in fact i still do not know what the word 'precoding' means , that is insane




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Ryanvuha/DRL_beam_hopping_precoding_GEO"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月17日(星期五) 下午4:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Ryanvuha/DRL_beam_hopping_precoding_GEO] Code question (Issue #1)





 
Hi, Have you solved these problems,and did your code run successfully?

**Comment by Fance-rest** [comment]
Joint_Precoding_and_Beam_Hoping_Design_for_Payload_Power_Minimization_in_GEO_Satellite_Communication_Networks
